app/library/postgres.py

```diff
@@ -224,6 +224,25 @@ async def get_scenario_by_friendly_name_from_postgres(
 
 
 async def delete_scenario_from_postgres(scenario_id: str, session):
+    # Get scene first
```

Comment:
This should be scenario, nice copy pasta 😛

Reply:
shoot LOL